packages/experiments-realm/ai-app-generator.gts

```diff
@@ -1,240 +1,66 @@
-import { type CardDef } from 'https://cardstack.com/base/card-api';
+import type { CardDef, CardContext } from 'https://cardstack.com/base/card-api';
 import {
   CardContainer,
   FieldContainer,
   BoxelInput,
   Button,
   TabbedHeader,
 } from '@cardstack/boxel-ui/components';
-import { eq } from '@cardstack/boxel-ui/helpers';
+import { and, bool, eq } from '@cardstack/boxel-ui/helpers';
+import { IconPlus } from '@cardstack/boxel-ui/icons';
 import { fn } from '@ember/helper';
 import { on } from '@ember/modifier';
 import { action } from '@ember/object';
 import GlimmerComponent from '@glimmer/component';
 import { tracked } from '@glimmer/tracking';
-import { AppCard, Tab } from './app-card';
+// @ts-ignore
```

Comment:
I see `packages/base/cards-grid.gts` has this import without an ignore, maybe there’s a `tsconfig.json` change or the like that would work for this?